src/resources/Entity/VerificationSessions.ts

```diff
@@ -0,0 +1,132 @@
+import Resource, { IResourceError, TResourceStatus } from '../../resource';
+import Configuration from '../../configuration';
+
+export const EntityVerificationSessionTypes = {
+  sms: 'sms',
+  sna: 'sna',
+  byo_sms: 'byo_sms',
+  byo_kyc: 'byo_kyc',
+  kba: 'kba',
+  auth_element: 'auth_element',
```

Comment:
I _think_ this should be just `element`

Reply:
yep, you are correct. i've only seen `element` returned in v2, not `auth_element`